casts/free-podcasts-screencasts-es.md

```diff
@@ -9,6 +10,11 @@
 * [Variados](#variados)
 
 
+### Bases de Datos
+
+* [Conceptos fundamentales: Bases de datos relacionales](https://programacionymas.com/series/aprende-bases-de-datos) - Juan Ramos (screencast)
```

Comment:
Sería más razonable tener una playlist de YouTube?: https://www.youtube.com/playlist?list=PLzSFZWTjelbJ01UciHPAWTqUFWesoGr9A

Reply:
This way they don't get distracted by other videos or ads.
But we can replace with the playlist if you prefer. Just confirm please.

Reply:
Thanks for answering. Replace it, please

Reply:
Okay. I did rebase to prevent adding many commits.